### PR TITLE
add params

### PR DESCRIPTION
**Pull request by nlehuby**
Add a way to launch it on another region.

A lot of things still need to be "parametrized" but that's a start.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,3 @@
 __pycache__
 data/*
-!data/.placeholder
+!data/config.ini
```
Comment (prhod) [inline .gitignore]: I'm not sure having config file in the `data` directory. What would you think of having a `config` folder?

File: data/config.ini [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[geo]
+OSMFile = ghana-internal.osh.pbf
+MapCenter = 5.6204/-0.2125
+
+[temporal]
+StartDate = 2017-07-05
+EndDate = 2017-09-05
+DeltaDays = 10
```
Comment (prhod) [inline data/config.ini]: It was a Delta of 1 in the source code. Why changing to 10 here?

File: transit_to_gif.py [hunk 3]
```diff
@@ -295,7 +311,7 @@ def show_date_on_image(image_path, date_to_display, nb_stops, nb_routes):
 
 file_names = sorted((os.path.join(img_tmp_dir, fn) for fn in os.listdir(img_tmp_dir) if fn.endswith('.png')))
 
-with imageio.get_writer('data/Accra_Ghana_Transit_data_creation.gif', mode='I', duration=0.4) as writer:
+with imageio.get_writer('data/output.gif', mode='I', duration=0.4) as writer:
```
Comment (prhod) [inline transit_to_gif.py]: IMHO having always the same `output` name. Can you add a parameter with the file name?

**Review by prhod — CHANGES_REQUESTED** [review]
Nice idea to extract those params. We may need to rename the repo if it's not only for Accra -_^

File: data/config.ini (hunk 2 above)
Comment (nlehuby) [reply data/config.ini]: my bad, it was to test quicker.

**Review by prhod — APPROVED** [review]
thanks for the fixes :+1:

**Pull request merged**